skietbaan_fe/src/bootstrap/NavbarMenuStyle.css

```diff
@@ -33,8 +38,13 @@
   }
 
   .profile-icon-grey-v2{
```

Comment:
`.notifications-icon-grey-v2` and `.notifications-icon-grey` and `profile-icon-grey-v2` are the same

Reply:
removed it because i was not using it